packages/create-novu-app/templates/app-react-email/ts/README-template.md

```diff
@@ -34,7 +34,7 @@ bun next-dev
 
 ## Your first workflow
 
-Your first email workflow can be edited in `./app/echo/client.ts`. You can adjust your workflow to your liking.
+Your first email workflow can be edited in `./app/workflow/myWorkflow.ts`. You can adjust your workflow to your liking.
```

Comment:
this path is up for debate, let me hear if we want it named some other way

Reply:
I think we should keep everything under a `novu` subpath, keeping in mind that we want to keep things simple but opinionated, so we can have the following:
```suggestion
Your first email workflow can be edited in `./app/novu/workflows.ts`. You can adjust your workflow to your liking.
```